src/pages/DetailsPage.js

```diff
@@ -126,7 +128,7 @@ const DetailsPage = ({
                                     </Text>
                                 </View>
                             ) : null}
-                            {details.timezone ? (
+                            { shouldShowLocalTime && details.timezone ? (
```

Comment:
There is an extra space before `shouldShowLocalTime`

Reply:
Removed